src/main/kotlin/com/chattriggers/ctjs/launch/CTJavaObjectMappingProvider.kt

```diff
@@ -11,10 +11,22 @@ object CTJavaObjectMappingProvider : JavaObjectMappingProvider {
         includeProtected: Boolean,
         includePrivate: Boolean
     ) {
+        val queue = ArrayDeque<Class<*>>()
         var current: Class<*>? = clazz
+
         while (current != null) {
             findRemappedMethods(current, map, includeProtected, includePrivate)
-            current = current.superclass
+            if (current.superclass != null && !current.superclass.name.startsWith("java.")) {
```

Comment:
I think it'd be better to check if the class is in a mapped package since there are only two. Maybe add a method to `Mappings` that checks the class against it's mapped classes list.